python/lsst/cp/verify/verifyFlat.py

```diff
@@ -0,0 +1,199 @@
+            Dictionary of measured statistics.  The inner dictionary
+            should have keys that are statistic names (`str`) with
+            values that are some sort of scalar (`int` or `float` are
+            the mostly likely types).
+
+        Returns
+        -------
+        outputStatistics : `dict` [`str`, `dict` [`str`, `bool`]]
+            A dictionary indexed by the amplifier name, containing
+            dictionaries of the verification criteria.
+        success : `bool`
+            A boolean indicating if all tests have passed.
+        """
+        ampStats = statisticsDict['AMP']
+        verifyStats = {}
+        success = True
+        for ampName, stats in ampStats.items():
+            verify = {}
+
+            # DMTN-101 Test 10.X: confirm that per-amplifier scatter is
+            #                     consistent with Poissonian
+            verify['NOISE'] = bool(np.abs(stats['NOISE']) <= np.sqrt(stats['MEAN']))
+
+            verify['SUCCESS'] = bool(np.all(list(verify.values())))
+            if verify['SUCCESS'] is False:
+                success = False
+
+            verifyStats[ampName] = verify
+
+        verifyDet = {}
+        detStats = statisticsDict['DET']
+
+        # DMTN-101 Test 10.Y: confirm intra-chip scatter is small.
+        verifyDet['SCATTER'] = bool(detStats['SCATTER']/detStats['MEAN'] <= 0.05)
+
+        verifyDet['SUCCESS'] = bool(np.all(list(verifyDet.values())))
+        if verifyDet['SUCCESS'] is False:
+            success = False
+
+        return {'AMP': verifyStats, 'DET': verifyDet}, bool(success)
+
+
+class CpVerifyFlatExpMergeConfig(CpVerifyExpMergeConfig):
+    """Inherits from base CpVerifyExpMergeConfig
+    """
+
+    def setDefaults(self):
+        super().setDefaults()
+        self.exposureStatKeywords = {'EXPOSURE_SCATTER': 'STDEV',  # noqa F841
+                                     }
+
+
+class CpVerifyFlatExpMergeTask(CpVerifyExpMergeTask):
+    """Inherits from base CpVerifyExpMergeTask
+    """
+
+    def exposureStatistics(self, statisticsDictionary):
+        """Calculate exposure level statistics based on the existing
+        per-amplifier and per-detector measurements.
+
+        Parameters
+        ----------
+        statisticsDictionary : `dict [`str`, `dict` [`str`, scalar]],
+            Dictionary of measured statistics.  The top level
+            dictionary is keyed on the detector names, and contains
+            the measured statistics from the per-detector
+            measurements.
+
+        Returns
+        -------
+        outputStatistics : `dict` [`str, scalar]
+            A dictionary of the statistics measured and their values.
+        """
+        detectorMeans = []
+        for detName, stats in statisticsDictionary.items():
+            # Get detector stats:
+            detectorMeans.append(stats['DET']['MEAN'])
+
+        return {'SCATTER': np.stdev(detectorMeans)}
```

Comment:
why do we use NumPy and not afw? It doesn't matter?

Reply:
I didn't think there'd be any difference.  This is a straight standard deviation, so they should be algorithmically identical (no clipping or masking needed).